src/main.rs

```diff
@@ -104,6 +105,8 @@ fn main_loop(mut surface: GL33Surface) {
             (input.value(input::Name::Up) - input.value(input::Name::Down)) * delta,
         );
 
+        renderer.reload();
```

Comment:
Not sure if this is the best place to do this on... Feels like there can be a lot of input lag if a whole file is loaded in a frame.

Reply:
It should definitely be done on a different thread. For now, what do you think about 7c18e91? I think it's good enough, in case it takes a while to get to #21.